### DISCUSSION TITLE
how can I purge a dataplane node’s cache use contolplane admin ApI?

### DISCUSSION THREAD

**Discussion by zcong2333** [Kong/kong#7842] [Help]
in kong hybrid mode,which admin api I can manage a dataplane node's lua_share_dict cache,is control node admin api or dataplane node itself,but dataplane dosen't listen 8001?

use control node admin api
curl '127.0.0.1:8001/cache/cache_key' 
{"message":"Not found"}

**Comment by dndx** [comment]
This is not something you can do on Hybrid mode. In general the DP cache is purged during config pushes anyway so it should not be as needed compared to traditional mode.
